packages/main/src/ToolbarButton.ts

```diff
@@ -180,17 +181,21 @@ class ToolbarButton extends ToolbarItem {
 	 * @type {string}
 	 * @name sap.ui.webc.main.ToolbarButton.prototype.text
 	 */
-	@property({ type: String })
+	@property()
 	text!: string;
 
 	/**
-	 * Button width
+	 * Defines the width of the button.
+	 * <br><br>
+	 *
+	 * <b>Note:</b> all CSS sizes are supported - 'percentage', 'px', 'rem', 'auto', etc.
+	 *
 	 * @name sap.ui.webc.main.ToolbarButton.prototype.width
 	 * @defaultvalue ""
 	 * @type {string}
 	 * @public
 	 */
-	@property({ type: String })
+	@property()
```

Comment:
I think we should return the old `CSSSize` type, but let's discuss it on one of the next meetings. String is fine for the moment.